src/config/routes.ts

```diff
@@ -1,23 +1,41 @@
-export const ROUTES = {
-  // Auth Stack
-  LOGIN: "Login",
-  MAIN_TABS: "MainTabs",
+export const ROOT_NAVIGATOR_ROUTES = {
+  AUTH_STACK: "AuthStack",
+  MAIN_TAB_STACK: "MainTabStack",
+} as const;
 
-  // Tab Stack
+export const MAIN_TAB_ROUTES = {
```

Comment:
I think we could leave only `History, Home and Discovery` tabs as we have on Figma right now.